exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestClassTransformation.java

```diff
@@ -68,7 +69,7 @@ public void testJaninoClassCompiler() throws Exception {
   @Test
   public void testJDKClassCompiler() throws Exception {
     logger.debug("Testing JDKClassCompiler");
-    sessionOptions.setOption(OptionValue.createString(OptionType.SESSION, ClassCompilerSelector.JAVA_COMPILER_OPTION, ClassCompilerSelector.CompilerPolicy.JDK.name()));
+    sessionOptions.setOption(OptionValue.createString(OptionType.SESSION, ClassCompilerSelector.JAVA_COMPILER_OPTION, ClassCompilerSelector.CompilerPolicy.JDK.name(), OptionScope.SESSION));
```

Comment:
Fallback option manager actually implements setOption and session option manager which is a subclass of the fallback option managerdoesn't override that. Moreover, we are creating the option value and passing it. So we need to have a new method to set option in session option manager. Or can we have a variable "type" and set it to OptionType and set it ?